cg/meta/upload/scout/balsamic_config_builder.py

```diff
@@ -32,32 +34,32 @@ def include_case_files(self):
         self.include_delivery_report()
 
     def include_sample_files(self, config_sample: ScoutBalsamicIndividual) -> None:
-        LOG.info("Including BALSAMIC specific sample level files")
+        LOG.info("Including BALSAMIC specific sample level files.")
 
         sample_id: str = config_sample.sample_id
         if config_sample.alignment_path:
-            if "tumor" in config_sample.alignment_path:
-                sample_id = "tumor"
-            elif "normal" in config_sample.alignment_path:
-                sample_id = "normal"
+            if SampleType.TUMOR in config_sample.alignment_path:
+                sample_id = SampleType.TUMOR.value
+            elif SampleType.NORMAL in config_sample.alignment_path:
+                sample_id = SampleType.NORMAL.value
 
         config_sample.vcf2cytosure = self.fetch_sample_file(
             hk_tags=self.sample_tags.vcf2cytosure, sample_id=sample_id
         )
 
-    def build_config_sample(self, db_sample: models.FamilySample) -> ScoutBalsamicIndividual:
-        """Build a sample with balsamic specific information"""
+    def build_config_sample(self, family_sample: models.FamilySample) -> ScoutBalsamicIndividual:
```

Comment:
Can we use `case_sample`instead. family does not really make sense for a cancer analysis
```suggestion
    def build_config_sample(self, case_sample: FamilySample) -> ScoutBalsamicIndividual:
```